packages/space-opera/src/components/config/reducer.ts

```diff
@@ -95,49 +158,77 @@ export const getConfig = (state: State) =>
 export function configReducer(
     state: ModelViewerConfig = {}, action: Action): ModelViewerConfig {
   switch (action.type) {
-    case SET_CONFIG: {
+    case SET_CONFIG:
       return action.payload;
-    }
     case SET_REVEAL:
       return {...state, reveal: action.payload};
     case SET_POSTER:
-      return {
-        ...state, poster: action.payload
-      }
+      return {...state, poster: action.payload};
```

Comment:
I like the expansion syntax, not sure if this was a thing the last time got into javascript.